example/src/main/java/com/vimeo/android/networking/example/vimeonetworking/TestAccountStore.java

```diff
@@ -33,24 +35,25 @@ public TestAccountStore(Context context) {
 //        mAccountManager = AccountManager.get(context);
     }
 
+    @Nullable
     @Override
     public VimeoAccount loadAccount() {
         return AccountPreferenceManager.getClientAccount();
     }
 
-    @Deprecated
     @Override
-    public void saveAccount(VimeoAccount vimeoAccount, String email, String password) {
-        saveAccount(vimeoAccount, email);
+    public void saveNonUserAccount(@NonNull VimeoAccount vimeoAccount) {
+        AccountPreferenceManager.setClientAccount(vimeoAccount);
     }
 
     @Override
-    public void saveAccount(VimeoAccount vimeoAccount, String email) {
+    public void saveAuthenticatedUserAccount(@NonNull VimeoAccount vimeoAccount,
```

Comment:
I do not think we need to confirm that it is a user account because 1. this is a simple demo that is just meant to show that you can respond to these callbacks, and 2. Now that we have split the method into two, this one should always be representative of a user account.